ddtrace/contrib/django/middleware.py

```diff
@@ -116,6 +116,11 @@ def process_response(self, request, response):
         try:
             span = _get_req_span(request)
             if span:
+                if response.status_code < 400:
```

Comment:
A `404` must not be considered an error (is just an example). I'd say to use `< 500`.

Reply:
Yes, this makes sense, 400s should be expected behaviour.